### PR TITLE
Updated distribution package structure

### PR DESCRIPTION
**Pull request by alparslanavci**
The command line distribution had its own directory structure. But since we are installing command line using `hazelcast` keyword, it should install the regular Hazelcast distribution plus the command line contents. This PR make the distribution directory structure the same as regular Hazelcast distribution and adds command line libs and scripts accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,16 +155,16 @@ cd distro/
 make
 ```
 
-After running the `make` command successfully, navigate to `distro/build/dist/bin` folder and run the `./hz` script:  
+After running the `make` command successfully, navigate to `distro/build/dist/hazelcast-4.1-BETA-1/bin` folder and run the `./hz` script:
```
Comment (pivovarit) [inline README.md:158]: I guess we could use some placeholder to avoid a need to manually update a directory name after the version name changes

Comment (alparslanavci) [reply README.md:158]: Do you mean to get the version from somewhere (ex.: pom.xml) in readme, or just use a placeholder such as `distro/build/dist/hazelcast-[HZ_VERSION]/bin`? If you mean the first, I don't know any way to do it. Maybe we can add it to release workflow?

Comment (pivovarit) [reply README.md:158]: If it's just a README, we could simply go for `/dist/{hazelcast.version}/bin` or similar, I think

Comment (alparslanavci) [reply README.md]: done 👍

**Review by pivovarit — APPROVED** [review]

**Pull request merged**